.editorconfig

```diff
@@ -3,4 +3,5 @@
 end_of_line = LF
 indent_style = tab
 trim_trailing_whitespace = false
-insert_final_newline = true
\ No newline at end of file
+insert_final_newline = true
+indent_size = 2
```

Comment:
We use indent size of 4: https://github.com/canjs/can-zone/blob/master/.editorconfig#L5

Reply:
ok!